pylearn2/tests/test_monitor.py

```diff
@@ -215,9 +215,11 @@ def test_revisit():
     for mon_batch_size in xrange(BATCH_SIZE, MAX_BATCH_SIZE + 1,
             BATCH_SIZE_STRIDE):
         nums = [1, 3, int(num_examples / mon_batch_size), None]
-        for num_mon_batches in nums:
-            for mode in sorted(_iteration_schemes):
-
+        
+        for mode in sorted(_iteration_schemes):
+            if mode == 'even_sequences' and nums is not None:
+                continue
```

Comment:
Can you add a comment regarding the reason why your new iteration mode is skipped in this test?